### PR TITLE
Corrected buffer operator onError behaviour documentation

### PR DESCRIPTION
**Pull request by mariusznet**
#3560

### CODE REVIEW

File: src/main/java/rx/Observable.java [hunk 1]
```diff
@@ -3285,8 +3285,9 @@ public final Boolean call(T first, T second) {
     /**
      * Returns an Observable that emits buffers of items it collects from the source Observable. The resulting
      * Observable emits connected, non-overlapping buffers, each containing {@code count} items. When the source
-     * Observable completes or encounters an error, the resulting Observable emits the current buffer and
-     * propagates the notification from the source Observable.
+     * Observable completes, the current buffer is emitted, and the event is propagated to all subscribed
```
Comment (akarnokd) [inline src/main/java/rx/Observable.java]: I don't like "propagate to all subscribed Observers". It suggests this is some hot operation and data is shared (such as in publish()).

**Comment by mariusznet** [comment]
Applied @akarnokd suggestion.

**Comment by stevegury** [comment]
:+1:

**Comment by akarnokd** [comment]
:+1:

**Pull request merged**